code/ASEParser.cpp

```diff
@@ -143,9 +143,9 @@ void Parser::LogWarning(const char* szWarn)
 
 	char szTemp[1024];
 #if _MSC_VER >= 1400
-	sprintf_s(szTemp,"Line %i: %s",iLineNumber,szWarn);

```

Comment:
Why do we use the "i" prefix for unsigned ints? There's some pretty funky hybrid Hungarian notation going on in some of these files, seems more confusing than anything else (perhaps excepting pointers?).